corehq/apps/fixtures/tests/test_upload.py

```diff
@@ -1,24 +1,36 @@
+from collections import namedtuple
 import os
 from django.test import SimpleTestCase
 from corehq.apps.fixtures.exceptions import FixtureUploadError
 from corehq.apps.fixtures.upload import FixtureWorkbook
 
-dot = os.path.dirname(__file__)
 
-UPLOAD_FILE = os.path.join(dot, 'test_upload', 'incorrect_fixtures.xlsx')
-ERRORS = [
+def make_path(args):
+    return os.path.join(os.path.dirname(__file__), *args)
+
+
+class UploadTestConfig(namedtuple('UploadTestConfig', ['upload_file_', 'error_messages'])):
```

Comment:
`namedtuple` doesn't let you prefix a value with `_` so I did this for speed while I was iterating on it, though I don't love it as a convention and eventually changed it.